Common++/header/MacAddress.h

```diff
@@ -27,34 +29,37 @@ namespace pcpp
 	 */
 	class MacAddress
 	{
-	public:
+	  public:
 		/**
 		 * Default constructor for this class.
 		 * Initializes object to me MacAddress::Zero
 		 */
 		MacAddress() : m_IsValid(true) { memset(m_Address, 0, sizeof(m_Address)); }
 
 		/**
-		 * A constructor that creates an instance of the class out of a byte array. The byte array length must be equal or greater to 6
-		 * (as MAC address is 6-byte long)
-		 * @todo there is no verification array length >= 6. If this is not the case, address will read uninitialized memory
+		 * A constructor that creates an instance of the class out of a byte array. The byte array length must be equal
+		 * or greater to 6 (as MAC address is 6-byte long)
+		 * @todo there is no verification array length >= 6. If this is not the case, address will read uninitialized
+		 * memory
 		 * @param[in] addr A pointer to the byte array containing 6 bytes representing the MAC address
 		 */
-		MacAddress(const uint8_t* addr) : m_IsValid(true) { memcpy(m_Address, addr, sizeof(m_Address)); }
+		MacAddress(const uint8_t *addr) : m_IsValid(true) { memcpy(m_Address, addr, sizeof(m_Address)); }
```

Comment:
Not very important but since PR already has many changes cleaning this type of unnecessary changes might be better before merging to ease review, track history etc.

PS: Shared library support for windows would be great.

Reply:
Thank you, I have corrected it.